Stockchecker.js

```diff
@@ -84,4 +83,4 @@ function doesntexist(element, name) {
 
 
 
-console.log(doesntexist(product, 'spoon'));
\ No newline at end of file
+console.log(doesntexist(product, 'Table'));
```

Comment:
when naming variables, if it is an array, it's more descriptive to name it in plural; products not product.